### PR TITLE
Move back to centos go and disable gofmt check in coverage job

### PR DESCRIPTION
**Pull request by alexeykazakov**

### CODE REVIEW

**Comment by alien-ike** [comment]
### Ike Plugins (test-keeper)

Thank you @alexeykazakov for this contribution!



It appears that no tests have been added or updated in this PR.

Automated tests give us confidence in shipping reliable software. Please add some as part of this change.

If you are an admin or the reviewer of this PR and you are sure that no test is needed then you can use the command `/ok-without-tests` as a comment to make the status green.


Your plugin configuration is stored in the [file](https://github.com/fabric8-services/fabric8-auth/blob/1d0f262ba42473aad682a52a2e687af50f0242a3/.ike-prow/test-keeper.yaml).

File: cico_setup.sh [hunk 1]
```diff
@@ -142,3 +142,16 @@ function cico_setup() {
...
   prepare;
 }
+
+function cico_setup_covarage() {
+  load_jenkins_vars;
+  install_deps;
+
+  # prepare() without check-go-format
+  make docker-start
+  # make docker-check-go-format
```
Comment (sbose78) [inline cico_setup.sh]: Fair enough.

File: cico_setup.sh [hunk 2]
```diff
@@ -142,3 +142,16 @@ function cico_setup() {
   install_deps;
   prepare;
 }
+
+function cico_setup_covarage() {
```
Comment (sbose78) [inline cico_setup.sh]: typo.

Comment (alexeykazakov) [reply cico_setup.sh]: Fixed

**Pull request merged**